### PR TITLE
Disable / enable swipe depending on screen.

### PR DESCRIPTION
**Pull request by cmpayc**
Fix react-community#1760

Problem: if you use the StackNavigator in the TabNavigator and you are at that time on the StackNavigator screen, the swipe will switch the screens to the TabNavigator.

Solution:
Adds a new option in the ```Screen Navigation Options``` in ```TabNavigator```:

**```swipeEnabled ```**
True or false to enable or disable swiping between tabs, if not set then defaults to TabNavigatorConfig option swipeEnabled .

### CODE REVIEW

File: src/views/TabView/TabView.js [hunk 1]
```diff
@@ -185,6 +184,9 @@ class TabView extends React.PureComponent<Props> {
 
     const tabBarVisible =
       options.tabBarVisible == null ? true : options.tabBarVisible;
+      
+    const swipeEnabled =
+      options.swipeEnabled == null ? this.props.swipeEnabled : options.swipeEnabled;
```
Comment (spencercarli) [inline src/views/TabView/TabView.js]: `swipeEnabled` should have a default prop set to `true` so that this wouldn't be a breaking change.

Comment (cmpayc) [reply src/views/TabView/TabView.js]: The value of ```props.swipeEnabled``` is taken from the TabNavigator.js, where it is already set to Presets value (for Android is ```true```, but for iOS is ```false```)

Comment (spencercarli) [reply src/views/TabView/TabView.js]: Ah okay great! I had confused myself with the `tabBarVisible` above. Once the docs are added I will merge. Thank you!

Comment (cmpayc) [reply src/views/TabView/TabView.js]: Adds a new option in the ```Screen Navigation Options``` in ```TabNavigator```:

**```swipeEnabled ```**
True or false to enable or disable swiping between tabs, if not set then defaults to TabNavigatorConfig option swipeEnabled .

**Comment by nicholasc** [comment]
Is this on hold because of documentation? If so, I am willing to do that.

**Comment by cmpayc** [comment]
Documentation updated. CircleCI errors fixed.

**Comment by spencercarli** [comment]
Thanks!

**Comment by nicholasc** [comment]
Is this going to be part of this week's release or next one?

**Comment by spencercarli** [comment]
It will be part of the next release, which should happen sometime next week.

**Comment by StepanNaryshkov** [comment]
I can not find any examples in the documentation. Can anyone show me how to use this feature or link to the documentation where is this feature?

**Comment by StepanNaryshkov** [comment]
@cmpayc Does this feature disable all my StackNavigator ? For example, I want to use swipe only tabs and if I click on a screen on a tab I want to disable it, it is possible?
Tab(swiped)>mainScreen(swiped)>child of mainScreen(disabled)

**Pull request merged**